ui/src/partials/navigation-tree.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if items}}
 <ol class="nav-list">
   {{#each items}}
-  <li class="nav-li" data-depth="{{#if ../level}}{{../level}}{{else}}0{{/if}}" {{#if currentPath}}data-state="expanded"
+  <li class="nav-li" {{#if currentPath}}data-state="expanded"
     {{/if}}>
```

Comment:
Can you join these two lines? Let's try and improve everything possible.

Reply:
sure